File: Regular/26/26-A.rb
#AtCorder Regular 26 A
input = STDIN.gets.chomp.split
problem = input[0].to_i
a = input[1].to_i
b = input[2].to_i

if problem <= 5 then
    puts problem * b
else
    puts 5 * b + ( problem - 5 ) * a
end

File: Regular/26/26-A.c
//AtCoder Regular 26 A
#include <stdio.h>

int main( void ){
    int problem;
    int a;
    int b;
    int times;

    scanf( "%d %d %d" , &problem , &a , &b );

    if( problem <= 5 ){
        times = problem * b;
    }
    else{
        times = 5 * b + ( problem - 5 ) * a;
    }

    printf( "%d\n" , times );

    return 0;
}
